torch/_dynamo/config.py

```diff
@@ -408,6 +408,10 @@ def default_debug_dir_root():
 # used for testing
 inject_BUILD_SET_unimplemented_TESTING_ONLY = False
 
+# How many iterations a for loop has to be before we decide to automatically
+# convert it to a function body.
+for_loop_medium_size_boundary = 100
+
```

Comment:
Please add a config option to disable this feature ... we can gradually turn it on.